### PR TITLE
Added defines for the return values of sendBytes.

### PR DESCRIPTION
**Pull request by Nicolasdejean**
[DO NOT MERGE] I didn't understand what you wanted to do with an enum ? #133

### CODE REVIEW

**Comment by FokkeZB** [comment]
Like [`ttn_fp_t`](https://github.com/TheThingsNetwork/arduino-device-lib/blob/8a96a4203c31fd016fb8737e92019e354e68de21/src/TheThingsNetwork.h#L34) this would assure the user that the function will only return variables from that type.

File: src/TheThingsNetwork.h [hunk 1]
```diff
@@ -15,10 +15,22 @@
 #define TTN_PWRIDX_868 "1"
 #define TTN_PWRIDX_915 "5"
 
+#define TTN_ERROR_SEND_COMMAND_FAILED (-1)
+#define TTN_ERROR_UNEXPECTED_RESPONSE (-10)
+#define TTN_SUCCESSFUL_TRANSMISSION 1
+#define TTN_SUCCESSFUL_RECEIVE 2
+
 #define TTN_BUFFER_SIZE 300
```
Comment (johanstokking) [inline src/TheThingsNetwork.h]: What's this?

File: src/TheThingsNetwork.h [hunk 2]
```diff
@@ -43,6 +55,7 @@ class TheThingsNetwork
     String model;
     airtime_t info;
     float airtime;
+    /* ttn_returnValue_t returnValue; */
```
Comment (johanstokking) [inline src/TheThingsNetwork.h]: Remove

File: src/TheThingsNetwork.h [hunk 3]
```diff
@@ -15,10 +15,22 @@
...
 #define TTN_BUFFER_SIZE 300
 
 typedef uint8_t port_t;
 
+/* enum ttn_returnValues_t { */
+/*   TTN_ERROR_SEND_COMMAND_FAILED, */
+/*   TTN_ERROR_UNEXPECTED_RESPONSE, */
+/*   TTN_SUCCESSFUL_TRANSMISSION, */
+/*   TTN_SUCCESSFUL_RECEIVE */
+/* }; */
```
Comment (johanstokking) [inline src/TheThingsNetwork.h]: Use the `enum` instead of the defines

File: src/TheThingsNetwork.h [hunk 4]
```diff
@@ -19,6 +19,13 @@
 
 typedef uint8_t port_t;
 
+enum ttn_returnValues_t {
```
Comment (johanstokking) [inline src/TheThingsNetwork.h]: Don't mix camelCasing and underscores. Make it `ttn_response_t`

**Comment by johanstokking** [comment]
@FokkeZB please test and merge

**Pull request merged**